core/src/main/java/com/bsarias/pokeapi/core/framework/navigation/InternalNavigation.kt

```diff
@@ -0,0 +1,7 @@
+package com.bsarias.pokeapi.core.framework.navigation
+
+object InternalNavigation {
+    const val DOMAIN = "myapp://"
+    const val LISTING = "${DOMAIN}listpokemon"
+    const val DETAILS = "${DOMAIN}details?pokemonName="
+}
```

Comment:
```suggestion
const val DOMAIN = "myapp://"
const val LISTING = "${DOMAIN}listpokemon"
const val DETAILS = "${DOMAIN}details?pokemonName="
    
```
La creación de un objeto en este caso no es necesaria y también genera un costo para el rendimiento:
https://bladecoder.medium.com/exploring-kotlins-hidden-costs-part-1-fbb9935d9b62